config/jobs/kubeflow/kubeflow-postsubmits.yaml

```diff
@@ -179,7 +179,7 @@ postsubmits:
       preset-service-account: "true"
     spec:
       containers:
-      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190214-f4092ae69-1.10
+      - image: gcr.io/k8s-testimages/kubekins-e2e:v20190219-caa48e6c3-1.10
```

Comment:
Don't we need to get rid/rotate the 1.10 configs?

Reply:
don't care about kubeflow